src/MSBuild.Conversion.Facts/MSBuildFacts.cs

```diff
@@ -296,8 +296,8 @@ public static class MSBuildFacts
         public const string PackagesSubstring = @"\packages";
         public const string NetStandard20 = "netstandard2.0";
         public const string NetCoreApp31 = "netcoreapp3.1";
-        public const string Net5 = "net5.0";
-        public const string Net5Windows = "net5.0-windows";
+        public const string Net5 = "net6.0";
```

Comment:
I don't know this code, but having a `const` named `Net5` with value `net6.0` looks misleading.

Reply:
I guess time to update to .NET 8.0 now.